app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,9 @@
+class FoodsController < ApplicationController
+
+  def index
+    @foods = FoodFacade.get_foods(params[:food])
```

Comment:
See my comments in your service and facade. How could we make a single call vs two?

Example:
```
def index
    results = FoodFacade.foods_by_keyword(params[:foods])
    @foods      = results[1]
    @total_hits = results[0]
end
```